app/router/newsletter.py

```diff
@@ -0,0 +1,46 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.responses import Response
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import SQLAlchemyError
+
+from app.config import get_settings, Settings
+from app.logger import log
+from app.database import get_db
+import app.schema as s
+import app.model as m
+
+newsletter_router = APIRouter(prefix="/newsletter", tags=["Newsletter"])
+
+
+@newsletter_router.post("/")
```

Comment:
I think it's better to add /subscription here, because we will have an unsubscribe later